### ISSUE TITLE
Impossible to toggle Other Calendars

### ISSUE THREAD

**Issue by sessionsdev** [YukiGasai/obsidian-google-calendar#130]
There are some external calendars in my google account that I subscribe to via URL.  I do not care about these calendars in obsidian, but I cannot find any way to exclude those calendars from timeline and schedule views.

**Comment by YukiGasai** [comment]
Can you add the calendars to the Blacklist at the end of the settings page?

**Comment by sessionsdev** [comment]
@YukiGasai No, it appears that other calenders do not appear in the drop down.   Attached is the calendar list from google, and the drop down from my obsidian.  Note that "Primary" and "Timeblocks" appears in the list, but "Work" does not.

![Screenshot 2023-06-02 at 10 05 54 AM](https://github.com/YukiGasai/obsidian-google-calendar/assets/15733809/742a1980-98b5-4fac-82a3-61c70988f986)

![Screenshot 2023-06-02 at 10 06 26 AM](https://github.com/YukiGasai/obsidian-google-calendar/assets/15733809/eaa3151e-0077-4e58-8e48-876626d66b6e)

**Comment by YukiGasai** [comment]
Sorry for the long delay.
I just can't reproduce this issue. With what option did you import the "Work" calendar into google.

**Comment by YukiGasai** [comment]
Closed due to inactivity.
